util/execdetails/execdetails.go

```diff
@@ -318,14 +320,23 @@ type RuntimeStatsColl struct {
 	readerStats map[string]*ReaderRuntimeStats
 }
 
+// concurrencyInfo is used to save the concurrency information of the executor operator
+type concurrencyInfo struct {
+	concurrencyName string
+	concurrencyNum  int
+}
+
 // RuntimeStats collects one executor's execution info.
 type RuntimeStats struct {
+	mu sync.Mutex
```

Comment:
Place `mu` just above the `concurency`, to show that `concurrency` is protected by `mu`.